1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,44 @@
 // Write your code here
+const items = document.querySelectorAll('img');
```

Comment:
because you are doing `querySelectorAll('img')` and getting all images, please rename `items` into `images` - this id gonna be much more self-descriptive name.